wmn-data.json

```diff
@@ -2217,14 +2227,19 @@
         "cat" : "coding"
        },
        {
-        "name" : "gitee",
-        "uri_check" : "https://gitee.com/{account}",
-        "e_code" : 200,
-        "e_string" : "Commits, issues, and pull requests will appear",
-        "m_string" : "Gitee is more than a development platform",
-        "m_code" : 404,
-        "known" : ["maxim"],
-        "cat" : "coding"
+        "name": "Gitee",
+        "uri_check" : "https://gitee.com/check",
+        "uri_pretty": "https://gitee.com/{account}",
+        "post_body": "do=user_username&val={account}",

```

Comment:
can you please check this as I'm not able to reproduce.

Reply:
Lol, they changed the registration system after the commit.